adala/runtimes/_openai.py

```diff
@@ -235,15 +185,10 @@ def execute(self, messages: List):
         if self.verbose:
             print(f"OpenAI request: {messages}")
 
-        if check_if_new_openai_version():
-            completion = self._client.chat.completions.create(
-                model=self.openai_model, messages=messages
-            )
-            completion_text = completion.choices[0].message.content
-        else:
-            # deprecated
-            completion = chat_completion_call(self.openai_model, messages)
-            completion_text = completion.choices[0]["message"]["content"]
+        completion = self._client.chat.completions.create(
+            model=self.openai_model, messages=messages
+        )
+        completion_text = completion.choices[0].message.content
```

Comment:
TY, this is much more readable now that it's not supporting both old and new openai formats